Sprint-3/3-mandatory-practice/implement/count.js

```diff
@@ -1,5 +1,23 @@
 function countChar(stringOfCharacters, findCharacter) {
-    return 5
+    let count = 0;  // start with zero, this helps to solve the issue of 
+    // non-occurence 
+
+    let i = 0; //where i is the index position indicating 
+    // we are starting from the beginning
+  
+    // The while loop goes through the string, one character at a time
+    // gets the character at the current position and
+    // checks if it matches the character we're looking for
+    while (i < stringOfCharacters.length) {
```

Comment:
This looks good - if there another type of loop we could use here? Potentially to remove the need for `i`?